### PR TITLE
backport: trivial backports April 6

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented
Trivial backports cherry-picked via script. 

## What was done?
Trivial backports

## How Has This Been Tested?
Light review and make -j8 check and functional tests all passed

## Breaking Changes
None that I saw

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/test/fuzz/process_message.cpp [hunk 1]
```diff
@@ -73,6 +73,10 @@ void fuzz_target(const std::vector<uint8_t>& buffer, const std::string& LIMIT_TO
         g_setup->m_node.peer_logic->ProcessMessage(p2p_node, random_message_type, random_bytes_data_stream, GetTimeMillis(), Params(), std::atomic<bool>{false});
     } catch (const std::ios_base::failure& e) {
     }
+    {
+        LOCK(p2p_node.cs_sendProcessing);
+        g_setup->m_node.peerman->SendMessages(&p2p_node);
```
Comment (knst) [inline src/test/fuzz/process_message.cpp:78]:
```
-g_setup->m_node.peerman->SendMessages(&p2p_node);
+g_setup->m_node.peer_logic->SendMessages(&p2p_node);
```

File: src/test/fuzz/process_messages.cpp [hunk 2]
```diff
@@ -72,6 +72,10 @@ FUZZ_TARGET_INIT(process_messages, initialize_process_messages)
             connman.ProcessMessagesOnce(random_node);
         } catch (const std::ios_base::failure&) {
         }
+        {
+            LOCK(random_node.cs_sendProcessing);
+            g_setup->m_node.peerman->SendMessages(&random_node);
```
Comment (knst) [inline src/test/fuzz/process_messages.cpp:77]:
```
-g_setup->m_node.peerman->SendMessages(&p2p_node);
+g_setup->m_node.peer_logic->SendMessages(&p2p_node);
```

**Review by knst — CHANGES_REQUESTED** [review]

File: src/test/fuzz/process_message.cpp (hunk 1 above)
Comment (knst) [reply src/test/fuzz/process_message.cpp:78]: due to missing 
```
    Merge #19791: [net processing] Move Misbehaving() to PeerManager
```

**Review by knst — APPROVED** [review]
LGTM now. utACK

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Pull request merged**